website/content/Resources/ip_acl_lists.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "ip_access_lists"
+date = 2020-07-31T23:34:03-04:00
+weight = 15
+chapter = false
+pre = ""
++++
+
+# databricks_ip_access_lists Resource
```

Comment:
`databricks_ip_access_list`

Reply:
sorry about that, I thought I deleted this file already.  Deleted on next commit.